tox.ini

```diff
@@ -56,8 +56,6 @@ deps =
     devdeps: git+https://github.com/bqplot/bqplot.git
     devdeps: git+https://github.com/bqplot/bqplot-gl.git
     devdeps: git+https://github.com/glue-viz/glue.git
-    # FIXME: https://github.com/spacetelescope/jdaviz/pull/2268
-    #devdeps: git+https://github.com/voila-dashboards/voila.git
```

Comment:
Do we need to replace this with a dev version of solara?

Reply:
I guess we probably should, although I don't think we have good test coverage of the standalone anyways that would catch much